docker-storage-setup.sh

```diff
@@ -1062,22 +1062,22 @@ setup_docker_root_dir() {
     return 1
   fi
 
-  [ -d "$DOCKER_ROOT_DIR" ] && return 0
+  [ -d "$ROOT_DIR" ] && return 0
 
   # Directory does not exist. Create one.
-  mkdir -p $DOCKER_ROOT_DIR
+  mkdir -p $ROOT_DIR
   return $?
 }
 
 reset_storage() {
-  
-    if [ "$DOCKER_ROOT_VOLUME" == "yes" ];then
+    [ "$ROOT_VOLUME" != "yes" ] && return 0
```

Comment:
Please remove this line. This is causing the tests to fail. Since the default value of `$DOCKER_ROOT_VOLUME` or `$ROOT_VOLUME` is `no`. This will just return and will never execute `reset_storage()`.